keanu-python/setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup
+
+setup(name='keanu',
+      version='0.1',
```

Comment:
@gordoncaleb might be worth talking about keanu-python version now before we merge this in.